src/editor/editor-mathfield.js

```diff
@@ -204,7 +204,7 @@ function MathField(element, config) {
 
     // Only display the virtual keyboard toggle if the virtual keyboard mode is
     // 'manual'
-    if (this.config.virtualKeyboardMode === 'manual') {
+    if (this.config.virtualKeyboardMode === 'manual' || this.config.showVirtualKeyboardButton) {
```

Comment:
@builtbywill so these configs already existed?

Reply:
@lily514 `isNativeMobileKeyboardEnabled` and `showVirtualKeyboardButton` are just a new properties I am checking for